src/python/tests_extended/test_dft_based.py

```diff
@@ -0,0 +1,662 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------------------------
+
+import os
+import platform
+import tempfile
+import unittest
+
+import numpy as np
+import pandas as pd
+from nimbusml import Pipeline
+from nimbusml.preprocessing.schema import ColumnSelector
+from nimbusml.preprocessing import ToString, ToKeyImputer, DateTimeSplitter
+from nimbusml.timeseries import TimeSeriesImputer, LagLeadOperator, RollingWindow, ForecastingPivot, ShortDrop
+from nimbusml.preprocessing import (TensorFlowScorer, FromKey, ToKey,
+                                    DateTimeSplitter, OnnxRunner)
+import onnxruntime as rt
+from data_frame_tool import DataFrameTool as DFT
+
+def get_tmp_file(suffix=None):
+    fd, file_name = tempfile.mkstemp(suffix=suffix)
+    fl = os.fdopen(fd, 'w')
+    fl.close()
+    return file_name
+
+class TestAutoMLTransforms(unittest.TestCase):
+    
+
+    def test_datetimesplitter(self):
+        df = pd.DataFrame(data=dict(
+            tokens1=[1, 2, 3, 157161600]
+        ))
+
+        cols_to_drop = [
+            'dtHour12', 'dtDayOfWeek', 'dtDayOfQuarter',
+            'dtDayOfYear', 'dtWeekOfMonth', 'dtQuarterOfYear',
+            'dtHalfOfYear', 'dtWeekIso', 'dtYearIso', 'dtMonthLabel',
+            'dtAmPmLabel', 'dtDayOfWeekLabel', 'dtIsPaidTimeOff','dtHolidayName'
+        ]
+
+        dts = DateTimeSplitter(prefix='dt') << 'tokens1'
+        xf = Pipeline([dts, ColumnSelector(drop_columns=cols_to_drop)])
```

Comment:
>ColumnSelector(drop_columns=cols_to_drop) [](start = 28, length = 41)

Why drop columns?

Reply:
This code is from the original test. I added `dtHolidayName` to the list because it's empty and `None` in ML.net and ORT and I cannot compare them.